@xen-orchestra/web-core/lib/components/tree/VtsTreeItem.vue

```diff
@@ -1,11 +1,13 @@
 <template>
-  <li class="vts-tree-item">
+  <li class="vts-tree-item" @click="handleClick">
```

Comment:
```suggestion
  <li class="vts-tree-item" @click="handleClick()">
```
[Here](https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/web-core/docs/guidelines/best-practices.md#component-must-use-inline-handler-or-inline-function-for-event-handling) is the explanation of why this change.